app/src/main/java/com/github/se/eventradar/ui/qrCode/QrCodeTicketUi.kt

```diff
@@ -58,8 +75,6 @@ fun QrCodeTicketUi(
 
   val qrScanUiState = viewModel.uiState.collectAsStateWithLifecycle()
 
-  val context = LocalContext.current
-
   ConstraintLayout(
       modifier = Modifier.fillMaxSize().testTag("qrCodeScannerScreen"),
   ) {
```

Comment:
If you think you have time, I would recommand that you use the newly made `AppScaffold` in `Reusable component`. You can see an exemple of how it works with tabs in `Home.kt`

Reply:
I really do not have time unfortunately. I have spent an easy 40+ hours this sprint on SWENT